webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -98,13 +96,24 @@ const TaskActionDropdown: React.FC<Props> = ({ task }: Props) => {
     // TODO show loading indicator when we have a button component that supports it.
   };
 
+  const menuItems: React.ReactNode[] = [];
+  if  (isResumable) menuItems.push(<Menu.Item key="activate">Activate</Menu.Item>);
+  if  (isPausable) menuItems.push(<Menu.Item key="pause">Pause</Menu.Item>);
+  if  (isArchivable) menuItems.push(<Menu.Item key="archive">Archive</Menu.Item>);
+  if  (isCancelable) menuItems.push(<Menu.Item key="cancel">Cancel</Menu.Item>);
+  if  (isKillable) menuItems.push(<Menu.Item key="kill">Kill</Menu.Item>);
+
+  if (menuItems.length === 0) {
+    return (
+      <div className={css.base} title="No applicable action items">
+        <Icon name="overflow-vertical" />
```

Comment:
thought: wrapping `Icon` with a disabled button to show the UI element just minus the dropdown menu.
```
<button disabled>
  <Icon name="overflow-vertical" />
</button>
```
May need additional styling applied to show the disabled state.

Reply:
yeah, I tried that will need to update the styles for it and give it a separate selector to avoid the onhover effect. I can add that in. should we go with a disabled cursor?